matrix/Matrix.hpp

```diff
@@ -290,8 +290,8 @@ class Matrix
 
     void print() const
     {
-        char buf[200];
-        write_string(buf, 200);
+        char buf[1000];
```

Comment:
It's only for debug prints but nevertheless I agree. What would be you proposition? Dynamic allocation?